app/controllers/welcome_controller.rb

```diff
@@ -1,3 +1,4 @@
 class WelcomeController < ApplicationController
-  def index; end
+  def index
```

Comment:
just for future reference, rubocop will prefer the one line syntax for empty methods

Reply:
Sure!